recipes/cadmesh/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "cadmesh" %}
+{% set version = "v0.1" %}
```

Conda versions shouldn't start with `v`:
```suggestion
{% set version = "0.1" %}
```
(I've suggested a change on the URL to match)